Add a mission transporting a group of Hai college students to Darkrest (#9601)
endless-sky/endless-sky: data/hai/hai missions.txt

			scene `thumbnail/hai pond strider`
			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
				decline

mission "Station Exploration 1"
	minor
	name "Bring students to <planet>"
	description "Bring a group of Hai and human students to <destination> to explore the abandoned station. They promise to pay you <payment>."
	passengers 5

> 	source "Greenwater"

Reviewer: ```suggestion
	"apparent payment" 8000
	source "Greenwater"
```
Now that the payment is missing.

	destination "Darkrest"
	to offer
		random < 30
		has "First Contact: Hai: offered"
	on offer
		conversation

> 			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`

Reviewer: ```suggestion
			`It is a sunny afternoon as you idly walk through the spaceport alone, until you somehow find yourself surrounded by a group of young humans and Hai.`
```
Did the idle walk become unidle when you get surrounded by the youngsters?

			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`
					decline
				`	"If anyone asks, I'll say I didn't know a permit was needed. Welcome aboard."`
			label accept
			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`

mission "Station Exploration 2"
	landing
	name "Bring students home to <planet>"
	description "Bring a group of Hai and human classmates back home to <destination>."
	passengers 5
	source "Darkrest"
	destination "Greenwater"
	to offer
		has "Station Exploration 1: done"
	on offer
		payment 8000
		conversation
			`Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>.`

> 			`	After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You then notice your passengers rushing towards your ship with a panicked look in their eyes. You imagine that they want to take advantage of that return trip you promised as soon as possible; however, disassociating yourself from them may save you some trouble with law enforcement.`

Reviewer: ```suggestion
			`	After passing the time in your ship for a few hours, you are startled by a loudspeaker sternly pronouncing something in Hai. You then notice your passengers rushing towards your ship with a panicked look in their eyes. You imagine that they want to take advantage of that return trip you promised as soon as possible; however, disassociating yourself from them may save you some trouble with law enforcement.`
```

			choice
				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
					to display
						has "visited planet: Darkrest"
				`	"Why do you want to go to <planet>?"`
					goto "question 1"
					to display
						not "visited planet: Darkrest"
				`	"Sure, I'm not hurting for cash right now."`
					goto accept
				`	"No way, that's too low. Find someone else."`
					decline
			label question
			`	"Well, that's kind of why we want to go there," says another one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
				goto continue
			label "question 1"
			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says another one of the Hai, "and we wanted to call his bluff."`
			label continue
			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`

> 			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`

Author: Ehh, I kind of like the abrupt, deadpan nature of the last sentence, which I feel is a bit lost with this suggestion.

			choice
				`	"Is this legal?"`
				`	"Not my business where you spend your vacations. Come aboard."`
					goto accept
			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`
					decline
				`	"If anyone asks, I'll say I didn't know a permit was needed. Welcome aboard."`
			label accept
			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`

> 	on complete
> 		payment 8000

Reviewer 2: Move this to the on offer of the next mission so that the text replacement in its conversation works.
```suggestion
```

mission "Station Exploration 2"

> 	priority

Reviewer 2: If a source has no spaceport, it needs to be a landing mission. Priority won't do anything for us here.
```suggestion
	landing
```

	name "Bring students home to <planet>"
	description "Bring a group of Hai and human classmates back home to <destination>."
	passengers 5
	source "Darkrest"
	destination "Greenwater"
	to offer
		has "Station Exploration 1: done"

> 		not "Station Exploration Abandon Ship: declined"

Reviewer 2: No longer needed.
```suggestion
```

> 	on offer

Reviewer 2: ```suggestion
	on offer
		payment 8000
```

			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though.`
			label trouble
			`	"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting.'" He chuckles as he leaves.`

> 				decline

Reviewer 2: No endpoint needed.
```suggestion
```

mission "Station Exploration Abandon Ship"
	invisible
	landing
	source

> 		attributes hai
> 		not attributes uninhabited

Reviewer 2: ```suggestion
		attributes "hai"
		not attributes "uninhabited"
```

	to offer

> 		has "Station Exploration 1: done"
> 		not "Station Exploration 2: done"

Reviewer 2: Needs updated offer conditions.
```suggestion
		has "Station Exploration 2: declined"
```

	on offer
		fine 70000
		conversation
			`Upon landing, you are accosted by the local police on <origin>. Behind your ship, you see your passengers disembarking from another Hai ship. Some of them are red-faced and crying, but they appear uninjured.`
			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
					goto questions
				`	"I'm innocent!"`
					goto normal
				`	(Run away.)`
			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`
			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
				goto normal
			label questions
			`	You can clearly tell that the officer has little patience for your questions. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`

> 			`	"You are being fined <payment> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`

Reviewer 2: This line is redundant with the last line in the normal label.
```suggestion
				goto fine
```

			label normal
			`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`

> 			`	"You are being fined <payment> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`

Reviewer 2: Use a `<fine>` replacement, not a `<payment>` replacement.
```suggestion
			label fine
			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
```

> 				decline

Reviewer 2: Don't need an endpoint; it's already the end of the conversation.
```suggestion
```

			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
			choice
				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
					goto question
				`	"Sure, I can take you there and back."`
					goto accept
				`	"How much can you pay me?"`
				`	"Sorry, I'm busy right now. Go ask some other captain."`
						decline
			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you! Is <payment> okay?" This last part is said in almost a whisper.`
			choice
				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
					goto question
				`	"Sure, I'm not hurting for cash right now."`
					goto accept
				`	"No way, that's too low. Find someone else."`
						decline
			label question

> 			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted call his bluff."`

Reviewer 3: ```suggestion
			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
```

			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
			choice
				`	"Is this legal?"`
				`	"Not my business where you spend your vacations. Come aboard."`
					goto accept
			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`
						decline
				`	"Not my business where you spend your vacations. Welcome aboard."`
			label accept
			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
				accept
	on visit

> 		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`

Reviewer 3: I think there is a generic version of this available as a phrase
You could have just launched and landed back bc you forgot smth and get this msg btw

Author: Made the phrase a little less generic.

mission "Station Exploration 2"
	name "Station Exploration"
	description "Bring a group of Hai and human classmates back home to <planet>."
	minor
	source "Darkrest"
	destination "Greenwater"
	passengers 5
	to offer
		has "Station Exploration 1: done"
	on offer
		conversation
			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`
				accept
	on visit
		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		conversation
			`Upon landing, you are accosted by local police.`
			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`

> 				`	"I'm innocent!"`

Reviewer 3: A run away option could be fun? Your call ofc idk much about missions

Author: Ehh, I don't really know how to write a run away option without it just ending up as just another 'you get shot and die lol' result

Reviewer 3: their guns do not kill, they only stun, so I was thinking you get shot, wake up later and then get a fine

Author: Added an option for this.

			`	"We just want to let you know what happened," he says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
			label trouble

> 				`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`

Reviewer 3: Idk why but in my head they were younger, probably bc you used classmates instead of students in the description?

				`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
					decline

mission "Abandon Ship"
	invisible
	landing
	source
		attributes hai
		not attributes uninhabited
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration 2: done"
	on offer

> 		payment -70000

Reviewer 3: You should probably make it a fine then if they're fining you (and that way if the player is broke it still works)

			scene `thumbnail/hai pond strider`
			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
				decline

mission "Station Exploration 1"
	name "Station Exploration"
	description "Bring a group of Hai and human classmates to <planet>, so that they can explore the abandoned station."
	minor
	source "Greenwater"
	destination "Darkrest"
	passengers 5
	to offer
		random < 30
		has "First Contact: Hai: offered"
	on offer
		conversation

> 			`	It is a sunny afternoon as you walk through the spaceport. As you idly inspect a fruit stand you somehow find yourself surrounded by a group of young humans and Hai.`

Reviewer: ```suggestion
			`It is a sunny afternoon as you walk through the spaceport. As you idly inspect a fruit stand you somehow find yourself surrounded by a group of young humans and Hai.`
```

			`	The students race onboard your ship, occasionally glancing back to make sure no one in the spaceport recognizes them.`
				accept
	on visit
		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 8000
		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker pronouncing something in Hai. You figure you should head back to the spaceport to see what's going on. Then again, abandoning them may save you some trouble with law enforcement.`

mission "Station Exploration 2"
	name "Station Exploration"
	description "Bring a group of Hai and human classmates back home to <planet>."
	minor
	source "Darkrest"
	destination "Greenwater"
	passengers 5
	to offer
		has "Station Exploration 1: done"
	on offer
		conversation

> 			`	The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`

Reviewer: ```suggestion
			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`
```

				accept
	on visit
		dialog `You look for your passengers, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		conversation

> 		`Upon landing, you are accosted by local police.`
> 		`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
> 		choice
> 			`	"Am I in trouble?"`
> 			`	"I'm innocent!"
> 		`	"We just want to let you know what happened," he says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
> 		`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. There's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
> 		choice
> 			`	"So, am I in trouble or not?"`
> 				goto trouble
> 			`	"Are my passengers going to be okay?"`
> 		`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't think their parents are going to be too thrilled with them, though."`
> 		label trouble
> 			`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
> 			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
> 				decline
> 

Reviewer: Missing indentations.
```suggestion
			`Upon landing, you are accosted by the local police.`
			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
				`	"I'm innocent!"
			`	"We just want to let you know what happened," he says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. There's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't think their parents are going to be too thrilled with them, though."`
			label trouble
			`	"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
				decline

```

mission "Abandon Ship"
	invisible
	landing
	source
		attributes hai
		not attributes uninhabited
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration 2: done"
	on offer
		payment -70000
		conversation

> 		`Upon landing, you are accosted by local police. Behind your ship, you see your passengers disembarking from a Hai ship. Some of them are crying, but they appear uninjured.`
> 		`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
> 		choice
> 			`	"Am I in trouble?"`
> 			`	"I'm innocent!"
> 		`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`
> 		`	"You are being fined <payment> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
> 			accept

Reviewer: Same here.
```suggestion
			`Upon landing, you are accosted by local police. Behind your ship, you see your passengers disembarking from a Hai ship. Some of them are crying, but they appear uninjured.`
			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
				`	"I'm innocent!"
			`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`
			`	"You are being fined <payment> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
				accept
```

			scene `thumbnail/hai pond strider`
			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
				decline

mission "Station Exploration 1"
	name "Station Exploration"

> 	description "Bring a group of Hai and human students to <planet>, so that they can explore the abandoned station."

Reviewer 4: ```suggestion
	description "Bring a group of Hai and human students to <planet> to explore the abandoned station."
```

mission "Station Exploration 2"
	name "Station Exploration"
	description "Bring a group of Hai and human classmates back home to <planet>."
	minor
	source "Darkrest"
	destination "Greenwater"
	passengers 5
	to offer
		has "Station Exploration 1: done"
	on offer
		conversation
			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		conversation
			`Upon landing, you are accosted by local police.`
			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice

> 				`	"Am I in trouble?"`

Reviewer 4: Shouldn't this go to normal?

Author: Wouldn't both `Am I in trouble?` and `I'm innocent!` go to normal, since they're both right above the goto?

Author: Nevermind I'm dumb.

			scene `thumbnail/hai pond strider`
			`Upon landing on <planet>, you receive a message from Osen. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoil the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
				decline

mission "Station Exploration 1"

> 	name "Station Exploration"

Reviewer: ```suggestion
	minor
	name "Bring students to <planet>"
```

> 	description "Bring a group of Hai and human students to <planet> to explore the abandoned station."

Reviewer: ```suggestion
	description "Bring a group of Hai and human students to <destination> to explore the abandoned station. They promise to pay you <payment>."
```

> 	minor
> 	source "Greenwater"
> 	destination "Darkrest"
> 	passengers 5

Reviewer: ```suggestion
	passengers 5
	source "Greenwater"
	destination "Darkrest"
```

	to offer
		random < 30
		has "First Contact: Hai: offered"
	on offer
		conversation
			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`
			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
			choice

> 				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
> 					goto question

Reviewer: If you've never been to Darkrest, you might want to add an additional option.
```suggestion
				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
					goto question
						to display
							has "visited planet: Darkrest"
				`	"Why do you want to go to <planet>?"`
					goto "question 1"
						to display
							not "visited planet: Darkrest"
```

				`	"Sure, I can take you there and back."`
					goto accept
				`	"How much can you pay me?"`
				`	"Sorry, I'm busy right now. Go ask some other captain."`
						decline
			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you! Is <payment> okay?" This last part is said in almost a whisper.`
			choice
				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
					goto question
				`	"Sure, I'm not hurting for cash right now."`
					goto accept
				`	"No way, that's too low. Find someone else."`
						decline
			label question

> 			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`

Reviewer: ```suggestion
			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
				goto continue
			label "question 1"
			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says one of the Hai, "and we wanted to call his bluff."`
			label continue
```

			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
			choice
				`	"Is this legal?"`
				`	"Not my business where you spend your vacations. Come aboard."`
					goto accept
			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`
						decline

> 				`	"Not my business where you spend your vacations. Welcome aboard."`

Reviewer: ```suggestion
				`	"Not my business where you spend your vacations. Come aboard."`
```

			label accept
			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 8000

> 		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should head back to the spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`

Reviewer: ```suggestion
		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should go back to <planet>'s spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`
```

> 		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should head back to the spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`

Reviewer: It might be a bit more effort, but if you had the Hai translator it would be interesting to hear the translation of the loudspeaker.

mission "Station Exploration 2"

> 	name "Station Exploration"

Reviewer: ```suggestion
	priority
	name "Bring students home to <planet>"
```
You want to make sure this always comes up after you complete Station Exploration 1.

> 	description "Bring a group of Hai and human classmates back home to <planet>."

Reviewer: ```suggestion
	description "Bring a group of Hai and human classmates back home to <destination>."
```

> 	minor
> 	source "Darkrest"
> 	destination "Greenwater"
> 	passengers 5

Reviewer: ```suggestion
	passengers 5
	source "Darkrest"
	destination "Greenwater"
```
I'm moving `minor` up the mission definition and changing it to `priority`.

	to offer

> 		has "Station Exploration 1: done"

Reviewer: ```suggestion
		has "Station Exploration 1: done"
		not "Station Exploration Abandon Ship: declined"
```
So that you don't get this mission after you've been told off.

	on offer
		conversation
			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		conversation

> 			`Upon landing, you are accosted by local police.`

Reviewer: ```suggestion
			`Upon landing, you are accosted by the local police on <planet>.`
```

			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
					goto normal
				`	"I'm innocent!"`
					goto normal
				`	(Run away.)`
			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`

> 			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him. Apparently you were not out for long.`

Reviewer: ```suggestion
			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
```

			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
			label trouble
				`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
				`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
					decline

> mission "Abandon Ship"

Reviewer: ```suggestion
mission "Station Exploration Abandon Ship"
```

	invisible
	landing
	source
		attributes hai
		not attributes uninhabited
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration 2: done"
	on offer
		fine 70000
		conversation

> 			`Upon landing, you are accosted by local police. Behind your ship, you see your passengers disembarking from a Hai ship. Some of them are crying, but they appear uninjured.`

Reviewer: ```suggestion
			`Upon landing, you are accosted by the local police on <origin>. Behind your ship, you see your passengers disembarking from another Hai ship. Some of them are crying, but they appear uninjured.`
```

			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
					goto normal
				`	"I'm innocent!"`
					goto normal
				`	(Run away.)`
			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`

> 			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him. Apparently you were not out for long.`

Reviewer: ```suggestion
			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
```

			label normal
			`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`
			`	"You are being fined <payment> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`

> 				accept

Reviewer: ```suggestion
				decline
```

	description "Bring a group of Hai and human students to <destination> to explore the abandoned station. They promise to pay you <payment>."
	passengers 5
	source "Greenwater"
	destination "Darkrest"
	to offer
		random < 30
		has "First Contact: Hai: offered"
	on offer
		conversation
			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`
			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
			choice
				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
					goto question
						to display
							has "visited planet: Darkrest"
				`	"Why do you want to go to <planet>?"`
					goto "question 1"
						to display
							not "visited planet: Darkrest"
				`	"Sure, I can take you there and back."`
					goto accept
				`	"How much can you pay me?"`
				`	"Sorry, I'm busy right now. Go ask some other captain."`
						decline

> 			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you! Is <payment> okay?" This last part is said in almost a whisper.`

Reviewer: ```suggestion
			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you!" The Hai pauses to glance at his friends and, in almost a whisper, says, "is <payment> okay?"`
```
I'm not sure who is talking at the moment, but maybe this works better.

			choice

> 				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`

Reviewer: ```suggestion
				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
					to display
						has "visited planet: Darkrest"
				`	"Why do you want to go to <planet>?"`
					goto "question 1"
						to display
							not "visited planet: Darkrest"
```
Same as above

Reviewer 2: The `to display` is indented too far.

> 					goto question

Reviewer: ```suggestion
```

				`	"Sure, I'm not hurting for cash right now."`
					goto accept
				`	"No way, that's too low. Find someone else."`

> 						decline

Reviewer: ```suggestion
					decline
```

			label question

> 			`	"Well, that's kind of why we want to go there," says one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
> 				goto continue
> 			label "question 1"
> 			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says one of the Hai, "and we wanted to call his bluff."`

Reviewer: ```suggestion
			`	"Well, that's kind of why we want to go there," says another one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
				goto continue
			label "question 1"
			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says another one of the Hai, "and we wanted to call his bluff."`
```

			label continue
			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`
			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`
			choice
				`	"Is this legal?"`
				`	"Not my business where you spend your vacations. Come aboard."`
					goto accept
			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`

> 						decline

Reviewer: ```suggestion
					decline
```

	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 8000
		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should go back to <planet>'s spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`

mission "Station Exploration 2"
	priority
	name "Bring students home to <planet>"
	description "Bring a group of Hai and human classmates back home to <destination>."
	passengers 5
	source "Darkrest"
	destination "Greenwater"
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration Abandon Ship: declined"
	on offer
		conversation

> 			`The spaceport is bathed in red light, and the booming voice continues to play. Your passengers rush back onto your ship without saying a word, and you take off before something worse happens.`

Reviewer: ```suggestion
			`The spaceport is bathed in red light, and the booming voice continues to play. Your red-faced passengers rush back onto your ship shouting at each other in a mixture of languages, but once they're through the airlock you take off before something worse happens.`
```
They rush back without saying a word? Seems like they should be clamouring with each other about who did what.

> 				accept

Reviewer: ```suggestion
			launch
```
Judging by the dialogue it should be launch rather than accept.

Reviewer 2: You're missing an indent.

			label normal
			`	"We just want to let you know what happened," the officer says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`
			label trouble

> 				`"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
> 				`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
> 					decline

Reviewer: ```suggestion
			`	"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`
			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`
				decline
```

mission "Station Exploration Abandon Ship"
	invisible
	landing
	source
		attributes hai
		not attributes uninhabited
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration 2: done"
	on offer
		fine 70000
		conversation

> 			`Upon landing, you are accosted by the local police on <origin>. Behind your ship, you see your passengers disembarking from another Hai ship. Some of them are crying, but they appear uninjured.`

Reviewer: ```suggestion
			`Upon landing, you are accosted by the local police on <origin>. Behind your ship, you see your passengers disembarking from another Hai ship. Some of them are red-faced and crying, but they appear uninjured.`
```

			`	"Captain <last>?", says a Hai officer. "Could you come with me for a moment?"`
			choice
				`	"Am I in trouble?"`
					goto normal
				`	"I'm innocent!"`
					goto normal
				`	(Run away.)`
			`	Almost as soon as you turn around to run you feel a sharp pain in your side as your consciousness fades away.`
			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
			label normal

> 			`	You can clearly tell that the officer has little patience for your antics. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries.`

Reviewer: It seems to me that asking whether you are in trouble or not isn't really an antic, although the officer should still be short on patience about the question. I tried writing something in a suggestion but I wasn't happy with it.

Author: Added a branch for if you asked if you were in trouble.

				decline

mission "Station Exploration 1"
	minor
	name "Bring students to <planet>"
	description "Bring a group of Hai and human students to <destination> to explore the abandoned station. They promise to pay you <payment>."
	passengers 5
	source "Greenwater"
	destination "Darkrest"
	to offer
		random < 30
		has "First Contact: Hai: offered"
	on offer
		conversation
			`It is a sunny afternoon as you idly walk through the spaceport, until you somehow find yourself surrounded by a group of young humans and Hai.`
			`	"You're a spaceship captain, right? Could you take us to <planet>?" says one of the Hai, as if he had practiced this line for a while now.`
			`	The human next to him gives him a nudge. "Don't forget to ask for a trip back, Tagreep, unless you want to be stranded there."`
			choice
				`	"What would you want to do on <planet>? Isn't that an abandoned station?"`
					goto question

> 						to display
> 							has "visited planet: Darkrest"

Reviewer 2: ```suggestion
					to display
						has "visited planet: Darkrest"
```

				`	"Why do you want to go to <planet>?"`
					goto "question 1"

> 						to display
> 							not "visited planet: Darkrest"

Reviewer 2: ```suggestion
					to display
						not "visited planet: Darkrest"
```

				`	"Sure, I can take you there and back."`
					goto accept
				`	"How much can you pay me?"`
				`	"Sorry, I'm busy right now. Go ask some other captain."`

> 						decline

Reviewer 2: ```suggestion
					decline
```

> 			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you!" The Hai pauses to glance at his friends and, in almost a whisper, says, "is <payment> okay?"`

Reviewer 2: ```suggestion
			`	The group starts to shuffle around with a nervous air. "Well, we're still in school, so money's kind of tight right now... But we can pay you!" The Hai pauses to glance at his friends and, in almost a whisper, says, "Is <payment> okay?"`
```

			choice
				`	"Why would you want to pay money to go to <planet>, anyhow? Isn't that an abandoned station?"`
					to display
						has "visited planet: Darkrest"
				`	"Why do you want to go to <planet>?"`
					goto "question 1"
					to display
						not "visited planet: Darkrest"
				`	"Sure, I'm not hurting for cash right now."`
					goto accept
				`	"No way, that's too low. Find someone else."`
					decline
			label question
			`	"Well, that's kind of why we want to go there," says another one of the Hai. "Some guy in our dorm told us that there's ghosts haunting <planet>, and we wanted to call his bluff."`
				goto continue
			label "question 1"
			`	"Some guy in our dorm told us that there's ghosts haunting <planet>," says another one of the Hai, "and we wanted to call his bluff."`
			label continue
			`	"We're planning to look around for an afternoon, see what we can find," one of the human students says, grinning. "Once we're done we'll let you know, and then we can leave."`

> 			`	One of the shorter Hai in the back pipes up. "I brought cameras for the trip!" He is ignored by the rest of the group.`

Reviewer 2: RIP this lil guy.

Author: ![image](https://github.com/endless-sky/endless-sky/assets/65418682/768b95a4-13e1-4816-94f1-0f2b69985e23)

			choice
				`	"Is this legal?"`
				`	"Not my business where you spend your vacations. Come aboard."`
					goto accept
			`	The students look at each other, considering this question for what is apparently the first time. "We've heard of other people going there with no problems, as long as they were... discreet about it," says one of the students, trying to assure both you and their fellow classmates. Another mentions having heard something about needing a permit to enter, but he is quickly shushed by the rest of the group.`
			choice
				`	"I'm not about to risk getting in trouble for this. Find someone else."`
					decline

> 				`	"Not my business where you spend your vacations. Come aboard."`

Reviewer 2: How about changing this one so it isn't a repeat of the answer from the choice above? Something like, "Well if anyone asks, I'll say I didn't know a permit was needed."

			label accept
			`	The students race onboard your ship. As they settle into their bunks, you can hear them speculating about what they will find on <planet>.`
				accept
	on visit
		dialog `You look for the group of students you were transporting, but realize that they took a ride on one of your escorts! Better depart and wait for your escorts to arrive in this star system.`
	on complete
		payment 8000

> 		dialog `Your passengers' journey mainly consisted of trying to scare each other by telling ghost stories and jumping out of dimly-lit corners of your ship. Despite these annoyances, you land at the station with no issues. Your passengers practically jump off your ship after hastily paying you <payment>. After passing the time in your ship for a few hours, you are shocked by a loudspeaker sternly pronouncing something in Hai. You figure you should go back to <planet>'s spaceport to see what's going on. Then again, disassociating yourself from them may save you some trouble with law enforcement.`

Reviewer 2: > You figure you should go back to `<planet>`'s spaceport to see what's going on.

Huh?
![image](https://github.com/endless-sky/endless-sky/assets/17688683/3e56ad9a-de95-43b7-b532-0eb790540185)

mission "Station Exploration 2"
	priority
	name "Bring students home to <planet>"
	description "Bring a group of Hai and human classmates back home to <destination>."
	passengers 5
	source "Darkrest"
	destination "Greenwater"
	to offer
		has "Station Exploration 1: done"
		not "Station Exploration Abandon Ship: declined"
	on offer
		conversation
			`The spaceport is bathed in red light, and the booming voice continues to play. Your red-faced passengers rush back onto your ship shouting at each other in a mixture of languages, but once they're through the airlock you take off before something worse happens.`

> 				launch

Reviewer 2: Darkrest has no spaceport. It seems like what you want is to take the on complete dialog from the first mission, make it a part of this on offer conversation, then give a choice to let the kids back in that either launches if you let them in or flees if you don't.

This also has no indication that you should return to Greenwater. Given that the abandon mission can be offered anywhere, should the conversation where you bring the kids with you also be offered anywhere? In order to do this, you could give this mission
```
to complete
	never
```
and have it be failed when you land anywhere in Hai space after this mission, which also triggers whichever mission corresponds to your decision.

Author: >Given that the abandon mission can be offered anywhere, should the conversation where you bring the kids with you also be offered anywhere?

You haven't really fulfilled your end of the bargain if you don't bring them back to Greenwater as you promised. I'd rather have it only trigger if you land there with the passengers.
I also added your suggestions in the first half of the comment, but I have no idea if I'm doing it correctly. I'm pretty terrible with mission syntax so I'd appreciate if you could give it a lookover and make sure nothing's broken.

			`	You look up, lying in the spot you just stood. Above you is the same officer which asked you to follow him, waving off another officer with a Pulse Rifle. Apparently you were not out for long.`
			label normal
			`	"We just want to let you know what happened," the officer says. He leads you away from your passengers, who are currently being examined by a medic for any potential injuries, and sits you down at a table nearby.`
			`	"I think you can tell by now that your passengers weren't exactly 'sanctioned visitors' to Darkrest. However, they didn't break anything - that announcement you heard was just a warning to refrain from touching a computer system without proper training, but it apparently spooked your passengers enough that they made a run for it. They're not in trouble, as there's no law against just wandering around a station as long as you're not destructive, but typically people who do so get permission first." He give a wry smile. "Saves us from having to deal with false alarms like this."`
			choice
				`	"So, am I in trouble or not?"`
					goto trouble
				`	"Are my passengers going to be okay?"`

> 			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though."`

Reviewer 2: ```suggestion
			`	"Yeah, they're fine. We're basically just here to make sure they didn't injure themselves. Don't imagine that their parents are going to be too thrilled with them, though.`
```

Reviewer 2: The next line starts with a quotation, though, and I imagine it's the same officer speaking both lines.

Author: Oh I see what you mean.

			label trouble
			`	"Simply bringing a paying passenger somewhere they requested to go isn't illegal. Your passengers were of age and knew what they were getting into, and you weren't deliberately enabling them to break the law. You're fine."`

> 			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting'." He chuckles as he leaves.`

Reviewer 2: ```suggestion
			`	The officer gets up. "Next time though, you should probably think twice before bringing some college kids somewhere to go 'ghost-hunting.'" He chuckles as he leaves.`
```

				accept
			label going
			`	"Thanks for your help, Captain. I'll contact you in a month once we're finished," says Likeet.`

> 				accept

Reviewer 2: ```suggestion
```

		payment 100000 50
		conversation
			`When you land on <planet>, you are approached by Likeet, who is carrying a large can of fake blood. "Thanks for the ride, Captain. Keep a lookout for my film in theaters. You'll know it when you see it." Once the cargo has been unloaded, she almost sprints away in excitement, no doubt trying to make it to her studio as quickly as possible. You notice that <payment> have been deposited into your account.`

> 

Reviewer 2: ```suggestion
```
